be/src/constants.ts

```diff
@@ -3,5 +3,7 @@ export const SEEK_API_BASE_URL = 'https://graphql.seek.com';
 export const SEEK_API_PATH = '/graphql';
 export const SEEK_API_URL = `${SEEK_API_BASE_URL}${SEEK_API_PATH}`;
 
+export const SEEK_BROWSER_TOKEN_PLAYGROUND_BASE_URL =
+  'https://indie-jessiej-public-test.public.indirect-apply.prod.outfra.xyz';
```

Comment:
😞

Reply:
Ahhh yes I forgot about that. We need to fix that.